### PR TITLE
Add IE converter based on Inspire Land Use

### PR DESCRIPTION
**Pull request by ivorbosloper**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -39,6 +39,9 @@ def get_description():
         # 'nl': [
         #     "pyogrio"
         # ]
+        "ie": [
+            "zipfile_deflate64"
```
Comment (m-mohr) [inline setup.py:43]: Shouldn't the package be `zipfile-deflate64`?

File: fiboa_cli/datasets/ie.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        "roles": ['producer', 'licensor']
+    }
+]
+ATTRIBUTION = "Ireland Department of Agriculture, Food and the Marine"
+LICENSE = {
+    "title": "This layer is published under the terms of the license Creative Commons Attribution 4.0 International (CC-BY-4.0)",
+    "href": "https://creativecommons.org/licenses/by/4.0/",
+    "type": "text/html",
+    "rel": "license"
+}
```
Comment (m-mohr) [inline fiboa_cli/datasets/ie.py:35]:
```suggestion
LICENSE = "CC-BY-4.0"
```

File: fiboa_cli/datasets/ie.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    "properties": {
+        "crop_name": {
+            "type": "string"
+        },
+    }
+}
+
+
+def migration(gdf):
+    gdf['crop_name'] = gdf['crop_name'].str.split(', ').str.get(0)
```
Comment (m-mohr) [inline fiboa_cli/datasets/ie.py:54]: Maybe a comment needed here, at least I'm wondering why we use only the part before the first comma...

Comment (ivorbosloper) [reply fiboa_cli/datasets/ie.py:44]: Yes, will do

File: fiboa_cli/convert_gml.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+from fiboa_cli import log
+import geopandas
+
+
+def gml_assure_columns(data, path, uri, layer, **kwargs):
+    # if GDAL opens a GML file, it generates a gfs file in which it tries to 'guess' a
+    # mapping from the GML XML file to features. This is not always correct.
+    # Call this function to add additional attributes from the GML
+    # We modify the GFS file, a more elegant solution is preferred
+    # See https://gdal.org/en/latest/drivers/vector/gml.html#schema for more info
```
Comment (m-mohr) [inline fiboa_cli/convert_gml.py:12]: I was wondering whether it makes more sense and is a bit cleaner to just ship pre-defined GFS files for the datasets?

Comment (ivorbosloper) [reply fiboa_cli/convert_gml.py:12]: I was wondering the same thing. I actually worked on a project that included a GFS file. After every base-data format change I was always struggling to update the GFS file. It took me a while to figure out this GFS file is generated. But I could document that too.

Comment (m-mohr) [reply fiboa_cli/convert_gml.py:12]: Okay, we can work on this in a separate PR.

Comment (m-mohr) [reply fiboa_cli/convert_gml.py:12]: See #104

**Review by m-mohr — APPROVED** [review]

**Pull request merged**